package.json

```diff
@@ -42,9 +42,9 @@
     "@babel/preset-react": "^7.0.0",
     "@babel/register": "^7.0.0",
     "@khanacademy/eslint-config": "0.0.5",
-    "@khanacademy/wonder-blocks-clickable": "^1.0.5",
-    "@khanacademy/wonder-blocks-color": "^1.1.10",
-    "@khanacademy/wonder-blocks-core": "^2.5.0",
+    "@khanacademy/wonder-blocks-clickable": "2.2.0",
+    "@khanacademy/wonder-blocks-color": "1.1.19",
+    "@khanacademy/wonder-blocks-core": "3.2.0",
```

Comment:
I dropped the `^` in these because `^2.2.0` ends up install `2.2.1` which has `wonder-blocks-core` 4.4.0 as a dep which then requires `flow-enum-runtime`.

Reply:
Wait, why is that a problem? Why can't we allow that?

Reply:
It felt a little weird since we aren't using clickable 2.2.1 or core 4.0.0 in webapp yet.  It's not really an issue though since we just need to make sure that `flow-enum-runtime` isn't include in the bundle.